app/models/github_repo.rb

```diff
@@ -0,0 +1,18 @@
+class GithubRepo
+
+	attr_reader :name, :url, :issues, :language
+
+	def initialize(attributes={})
+		@name 		= attributes[:name]
+		@url 			= attributes[:html_url]
```

Comment:
<img width="481" alt="screen shot 2017-01-31 at 4 04 43 pm" src="https://cloud.githubusercontent.com/assets/19585280/22488404/0ce66364-e7cf-11e6-8ecb-03b4695d7f5f.png">

@case-eee I don't understand why it shows up this way on github. This is what it looks like in my editor!